web/test/pages/profile/profile-foreign.test.tsx

```diff
@@ -282,6 +282,59 @@ describe("profile-foreign", () => {
       ).not.toBeInTheDocument();
     });
 
+    it("renders locked profile address fields when business has formed", () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          businessPersona: "FOREIGN",
+        }),
+        formationData: generateFormationData({
+          completedFilingPayment: true,
+          formationFormData: generateFormationFormData({
+            addressLine1: "123 Testing Road",
+            addressLine2: "",
+            addressCity: "New York",
+            addressState: {
+              name: "New York",
+              shortCode: "NY",
+            },
+            addressZipCode: "10030",
+            businessLocationType: "US",
+          }),
+        }),
+      });
+      renderPage({ business });
+
+      expect(screen.getByTestId("locked-profileAddressLine1")).toBeInTheDocument();
+      expect(screen.getByTestId("locked-profileAddressCityStateZip")).toBeInTheDocument();
+    });
+
+    it("does not render locked profile address fields when business has not been formed", () => {
```

Comment:
[sand] The goal of this test should be to assert that there are editable fields on the page, not just that we don't see the locked form correct?

```suggestion
    it("renders editable profile address fields when business has not been formed", () => {
```

We'll also want to validate that those editable fields are on the page.

Reply:
@AnJuHyppolite I'd like for us to address this still.